BigCommerce/UserInterface/01_Settings.md

```diff
@@ -0,0 +1,8 @@
+# Settings
+
+The *Settings* menu entry contains the following tabs relevant for the *BigCommerce* configuration:
+
+- [CONNECTIONS](../UserInterface/01a_Connections.md)
+- [VARIANT SETS](../../Channels/UserInterface/07b_VariantSets.md) (Reference to *Omni-Channel* documentation)
+
```

Comment:
Ist der Tab VARIANT SETStatsächlich da, wie in Omni-Channel? Gibt es keine anderen Tabs vorhanden (du schreibst "following tabs relevant for BigCommerce configuration, daher meine Frage). (Ich habe keinen Zugang auf die BigCommerce-Sandbox und kann das nicht prüfen).